lib/omnibus/git_cache.rb

```diff
@@ -32,7 +32,7 @@ class GitCache
     # will not have the generated content, so these snapshots would be
     # incompatible with the current omnibus codebase. Incrementing the serial
     # number ensures these old shapshots will not be used in subsequent builds.
-    SERIAL_NUMBER = 1
+    SERIAL_NUMBER = 2
```

Comment:
Is this because there could be cache snapshots with license collection errors in them?